constitution.tex

```diff
@@ -724,8 +724,6 @@
 \renewcommand{\theenumi}{\alph{enumi}} % For this section, we want items to use letters
 \begin{enumerate}
 	\item Candidates must have been an On-Floor Active Member in good standing within the last twelve months, or been an Off-Floor Active Member and have lived on the floor within the last twelve months.
```

Comment:
Spencer and I talked a few weeks ago and agreed that removing the RTP's ability to waive requirements would be reasonable if we adjusted the requirements to also be more reasonable. I suggested and we agreed upon the following change:
```suggestion
	\item Candidates must have been an Active Member within the last twelve months.
```